New actions in Palo Alto PAN-OS plugin (#799)
rapid7/insightconnect-plugins: palo_alto_pan_os/help.md

### Actions

#### Add Address Objects to Group

> This action adds an address objects to an address group. Supports IPv6.

Reviewer: ```suggestion
This action adds address objects to an address group. Supports IPv6.
```

#### Add Address Object to Group

> This action adds address objects to an address group. Supports IPv6.

Reviewer 2: ```suggestion
This action adds address objects to an address group. This action supports FQDNs, IPv4 and IPv6 addresses.
```

{
  "success": true,
  "address_objects": [
    "test.com",
    "domain.com",
    "198.51.100.102,
    "198.51.100.100",
    "198.51.100.101",
    "example.com"
  ]
}
```

#### Get Addresses from Group

> This action is used to get an addresses from an address group.

Reviewer: ```suggestion
This action is used to get addresses from an address group.
```

# Version History

> * 7.0.0 - New action Get Addresses from Group | Rename Add Address Object to Group action to Add Address Objects to Group and support adding a list of address objects

Reviewer: ```suggestion
* 7.0.0 - New action Get Addresses from Group | Rename 'Add Address Object to Group' action to 'Add Address Objects to Group' and support adding a list of address objects
```

# Version History

> * 6.1.0 - Improve error handling for xpath elements and paths in `pa_os_request.py` | New action `Get Addresses from Group` | Support adding a list of address objects in `Add Address Object to Group` action

Reviewer 2: ```suggestion
* 6.1.0 - Improve error handling for xpath elements and paths in `pa_os_request.py` | New action Get Addresses from Group | Support adding a list of address objects in Add Address Object to Group action
```
